README.md

```diff
@@ -122,6 +190,25 @@ to rollback its action - in this case the first argument will change as
 appropriate, so must be obeyed by the script (e.g. a "create" hook will be
 called with "destroy" if it has to be rolled back later).
 
+## Transactions
+
+Most hooks are triggered during database transaction. This can cause deadlocks
+when hook scripts emits database updates via Foreman CLI or API. It is
+recommended to avoid this behavior and write a Foreman Plugin instead.
```

Comment:
lower case "Plugin", and I don't think that's good advice, perhaps better to remove it.